fe-public/src/pages/blog/building-the-ie-website.md

```diff
@@ -1,9 +1,71 @@
 ---
 layout: '../../layouts/Blog.astro'
-title: Building the AI and Robotics Website
+title: Building a website for the Robotics and AI Club
 ---
 
-This blog highlights some insights from our path of building the IE website.
+# Unveiling the Secrets Behind the IE Robotics and AI Club: Math, Leaderboard Stack, and SEO/UI/UX Insights
 
-# Devlog
-After talking about the website with the team, we decided to use [Astro](https://astro.build/) as our static site generator. Astro is a new static site generator that uses React as its templating language. It also has a lot of cool features like partial hydration, which allows us to only hydrate the parts of the page that need to be interactive. This is great for performance and accessibility. We also talked about the internal website which will be built using Vue.js. We decided to use Vue.js because it is a framework that we are all familiar with and it is also very easy to use. We also decided to use TailwindCSS as our CSS framework because it is very easy to use and it is also very customizable.
+## Introduction
+- A brief overview of what readers can expect from the blog post.
+
+## The Math Behind the Waves and Word Cloud
+### By @i-be-keggles
+- Explain the mathematical models or algorithms driving the wave patterns and word cloud on the website.
+
+```code
+// Code snippet explaining math algorithms
+```
```

Comment:
@i-be-keggles Could you please write some details about how you made the wave? 🙏🏻

Reply:
Put some stuff down. Lmk what you think

Reply:
🪨